<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AboutMeDialog</class>
 <widget class="QDialog" name="AboutMeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>236</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>About me</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
   <item>
    <widget class="QLabel" name="labelTitle">
     <property name="text">
      <string>&lt;h1&gt;About me&lt;/h1&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelUserId">
       <property name="text">
        <string>User-ID:</string>
       </property>
       <property name="buddy">
        <cstring>spinBoxUserId</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelEmail">
       <property name="text">
        <string>E-Mail:</string>
       </property>
       <property name="buddy">
        <cstring>lineEditEmail</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelLongUsername">
       <property name="text">
        <string>Long username:</string>
       </property>
       <property name="buddy">
        <cstring>lineEditLongUsername</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelText">
       <property name="text">
        <string>Text:</string>
       </property>
       <property name="buddy">
        <cstring>lineEditText</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelUsername">
       <property name="text">
        <string>Username:</string>
       </property>
       <property name="buddy">
        <cstring>lineEditUsername</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="spinBoxUserId">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="maximum">
        <number>16777215</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditEmail">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEditLongUsername">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEditText">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEditUsername">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
